projects/GKCore/GKCore/BaseContext.cs

```diff
@@ -1467,7 +1467,11 @@ public string GetPrimaryBitmapUID(GDMIndividualRecord iRec)
 
         public bool IsUnknown()
         {
-            return string.IsNullOrEmpty(fFileName) || !File.Exists(fFileName);
+            var isNew = string.IsNullOrEmpty(fFileName) || !File.Exists(fFileName);
+            if (isNew) return true;
+
+            var ext = FileHelper.GetFileExtension(fFileName);
+            return ext != ".ged" && ext != ".geds";

```

Comment:
I chose to implement changes here, because I think this would be appropriate place. If you want, I can split this into another function, eg, calling it "CanSave" or similar.